raisinbread/RB_files/RB_mri_parameter_form_for_QC.sql

```diff
@@ -0,0 +1,41 @@
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
this file should not exist

Reply:
@ridz1208 Without this file, the QC will not load. Where should I put the file?

Reply:
I will look into this to understand it a bit more. thx